src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Values/CultureValue.cs

```diff
@@ -0,0 +1,43 @@
+using System.Globalization;
+using System.Text.Encodings.Web;
+using Fluid;
+using Fluid.Values;
+using OrchardCore.Localization;
+
+namespace OrchardCore.DisplayManagement.Liquid.Values;
+
+public class CultureValue : FluidValue
```

Comment:
```suggestion
private sealed class CultureValue : FluidValue
```

Reply:
Until someone complains then need it, and in the meantime they can create their own.